Cloud Application Manager/Monitoring/CTLCloudMonitoringUI.md

```diff
@@ -31,6 +31,11 @@ For servers that have 'metric' server checks applied to them, for which all Mana
 
 After logging in the user will be in their personal workspace. Click on the workspace switcher to choose the workspace of which agent(s) you wish to view.  
 
+#### [AWS Dashboards](https://www.ctl.io/knowledge-base/cloud-application-manager/monitoring/awsdashboards/)
```

Comment:
change link to `(awsdashboards.md)`

Reply:
I have made the updates to the events, policies, suppressions, and AWS Dashboards links.  I am not sure I did them right though.